VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/SettingsScreen.test.tsx

```diff
@@ -2,13 +2,11 @@ import 'react-native'
 import React from 'react'
 import { Share } from 'react-native'
 import { BIOMETRY_TYPE } from 'react-native-keychain'
-// Note: test renderer must be required after react-native.
-import { ReactTestInstance } from 'react-test-renderer'
-import { context, findByTestID, findByTypeWithText, mockNavProps, render, RenderAPI } from 'testUtils'
+
+import { fireEvent, screen } from '@testing-library/react-native'
```

Comment:
This can be grouped with the other node modules

Reply:
updated